cmd/avalanche.go

```diff
@@ -55,6 +55,7 @@ var (
 	remoteTenant         = kingpin.Flag("remote-tenant", "Tenant ID to include in remote_write send").Default("0").String()
 	tlsClientInsecure    = kingpin.Flag("tls-client-insecure", "Skip certificate check on tls connection").Default("false").Bool()
 	remoteTenantHeader   = kingpin.Flag("remote-tenant-header", "Tenant ID to include in remote_write send. The default, is the default tenant header expected by Cortex.").Default("X-Scope-OrgID").String()
+	outOfOrder           = kingpin.Flag("out-of-order", "Enable out-of-order timestamps in remote write requests").Default("true").Bool()
```

Comment:
Nice!

One idea, instead of a bool should we have a flag like `out-of-order.min-time` which would control the minimum time this can go back in time?

Reply:
It's fine to do it in another PR though, thanks!

Reply:
okkay! i will raise another PR for that. thank you !